src/Handler/NginxHandler.php

```diff
@@ -32,6 +38,11 @@ public function handleResult(DetectionResult $detectionResult): void
         try {
             $data = \json_decode($detectionResult->getData(), false, 512, JSON_THROW_ON_ERROR);
 
+            // Nginx 'default' sites should not be indexed.
+            if (str_ends_with($data->config, self::NGINX_DEFAULT)) {
```

Comment:
Are we should that the default domain never is used to serv anything?

Reply:
No. But I haven't seen any examples so far.